addons/godot-xr-tools/xr/start_xr.gd

```diff
@@ -48,13 +44,16 @@ var xr_interface : XRInterface
 ## XR active flag
 var xr_active : bool = false
 
-# Current refresh rate
-var _current_refresh_rate : float = 0
+## XR frame rate
+var xr_frame_rate : float = 0
+
+# Is a WebXR is_session_supported query running
+var _webxr_session_query : bool = false
 
 
 # Handle auto-initialization when ready
 func _ready() -> void:
-	if !Engine.is_editor_hint() and auto_initialize:
+	if !Engine.is_editor_hint():
```

Comment:
Seeing we decided to remove `auto_initialize`, we should rename our `initialize` function to `_initialize` to indicate it's private.
We should also add an `xr_failed_to_initialize` signal that we emit when `_initialize` fails so the user can react to this scenario, either by setting the game up for non-VR or to exit the app.
Similarly maybe we should also add an `xr_initialized` signal on successful initialisation for the same reason though that feels like overkill.

Reply:
I've added the signal and I trigger it if:
* OpenXR fails to initialize
* WebXR fails to initialize
* No XRInterface is found

Reply:
I've also renamed initialize to _initialize